docs/fundamentals/code-analysis/quality-rules/snippets/csharp/all-rules/ca2208.cs

```diff
@@ -2,6 +2,7 @@
 
 namespace ca2208
 {
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
```

Comment:
Oh interesting, for some reason, this is one rule we've explicitly turned on in the editorconfig: https://github.com/dotnet/docs/blob/266c099450b3b5433e04dbec73515ef1c9a330de/.editorconfig#L81. Actually there are new settings in .NET 6 where we can bulk enable security rules - I'll look into that.

Reply:
It is an error, I think that's what we wanted. This PR adds an explicit disable/restore to remove the error from the example that shows the issue.